internal/provider/schema_gitlab_release_link.go

```diff
@@ -0,0 +1,85 @@
+package provider
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/xanzy/go-gitlab"
+)
+
+func gitlabReleaseLinkGetSchema() map[string]*schema.Schema {
+	validLinkTypes := []string{"other", "runbook", "image", "package"}
+
+	return map[string]*schema.Schema{
+		"project": {
+			Description: "The ID or [URL-encoded path of the project](https://docs.gitlab.com/ee/api/index.html#namespaced-path-encoding).",
+			Type:        schema.TypeString,
+			ForceNew:    true,
+			Required:    true,
+		},
+		"tag_name": {
+			Description: "The tag associated with the Release.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"name": {
+			Description: "The name of the link. Link names must be unique within the release.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"url": {
+			Description: "The URL of the link. Link URLs must be unique within the release.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"filepath": {
+			Description: "Relatively path for a [Direct Asset link](https://docs.gitlab.com/ee/user/project/releases/index.html#permanent-links-to-release-assets).",
+			Type:        schema.TypeString,
+			Optional:    true,
+		},
+		"link_type": {
+			Description:      fmt.Sprintf("The type of the link: %s. Defaults to %s.", renderValueListForDocs(validLinkTypes), validLinkTypes[0]),
```

Comment:
Update slightly to match the verbiage we use for other similar lists 🙂 
```suggestion
			Description:      fmt.Sprintf("The type of the link. Valid values are %s. Defaults to %s.", renderValueListForDocs(validLinkTypes), validLinkTypes[0]),
```